### ISSUE TITLE
[Usable] 'Send' button is not accessible in scan mode. 

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1307]
**Actual Result:**
'Send' button is not accessible in scan mode.

**Expected Result:**
'Send' button should be accessible in scan mode.

**Repro Steps:**
1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Send' button icons present in bottom of screen.
3. Open Narrator (Using Ctrl + Windows +Enter).
4. Start Scan mode (Caps lock + Space).
5. Navigate to 'Send' button by using Up/Down arrow keys.
6. Observe that 'Send' button is not accessible in scan mode.

**User Experience:**
Users who rely on screen readers may get confused and face difficulty if navigation is not correct or logical and controls are not accessible in scan mode.

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[Usable_Send button not accessible in scan mode.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2547171/Usable_Send.button.not.accessible.in.scan.mode.zip)

**Comment by corinagum** [comment]
According to accessibility team, testing in scan mode is not part of our requirements. The button is accessible via tabbing.

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yUsable
